help_to_heat/portal/download_views.py

```diff
@@ -297,7 +405,89 @@ def add_extra_row_data(referral, exclude_pii=False):
         "submission_date": created_at.date(),
         "submission_time": created_at.time().strftime("%H:%M:%S"),
         "referral_id": referral.formatted_referral_id,
+        "user_selected_property_type": user_selected_property_type,
+    }
+
+    epc_data = row.get(epc_details_field) or {}
+
+    # main heat source used to come from a specific field in the user data
+    # now it is sent via the epc details object
+    # use the legacy field if available, else draw from EPC
+    property_main_heat_source = row.get(property_main_heat_source_field) or epc_data.get("mainheat-description")
```

Comment:
For what it's worth I think it equally raises questions we we now treat it differently here (not suggesting it's wrong necessarily, just that it's going to stick out either way)

Reply:
I agree, though I think it's more sensible to keep the higher traffic routing code saner and handle it here

does the comment explain this well enough?

Reply:
Yes think so